packages/flutter_tools/test/asset_bundle_package_test.dart

```diff
@@ -130,11 +154,9 @@ $assetsSection
         <String>['test_package'],
         expectedAssetManifest,
       );
-    }, overrides: <Type, Generator>{
-      FileSystem: () => new MemoryFileSystem(),
-    });
+    }, overrides: contextOverrides);
     
-    testUsingContext('One package with one asset not specified', () async {
+    testUsingContext("One asset is bundled when the package has one asset, listed in the app's pubspec", () async {
```

Comment:
nit: style guide says to use `'` rather that `"`.

Reply:
I used `"` to avoid escaping `'` in "app's". Is escaping preferred in this case?

Reply:
Ah, I see. Either's fine.

Reply:
Cool, thanks!